SimCalorimetry/EcalEBTrigPrimProducers/plugins/SealModules.cc

```diff
@@ -1,7 +1,9 @@
 #include "FWCore/Framework/interface/MakerMacros.h"
 
 #include "EcalEBTrigPrimProducer.h"
-#include "EcalEBTrigPrimAnalyzer.h"
+#include "EcalEBTrigPrimPhase2ESProducer.h"
+#include "EcalEBTrigPrimPhase2Producer.h"
 
 DEFINE_FWK_MODULE(EcalEBTrigPrimProducer);
-DEFINE_FWK_MODULE(EcalEBTrigPrimAnalyzer);
```

Comment:
Out of curiosity: why are you removing this? If the plugin is not needed any more, perhaps also its code could be removed from CMSSW

Reply:
@perrotta That is an old version of the Analyzer, which works with the TP version for Phase2 now in release since 2017. But would not work with the Phase2 TP I am trying to release here.  I would prefer to keep as a reference for the future if you do not  mind too much